modules/end-user-guide/pages/optional-parameters-for-the-urls-for-starting-a-new-workspace.adoc

```diff
@@ -0,0 +1,111 @@
+
+The URL for starting a new workspace supports concatenation of multiple URL parameters by using `&` with the following URL syntax:
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?__<url_parameter_1>__&__<url_parameter_2>__&__<url_parameter_3>__
+----
+
+== URL parameter for the workspace IDE
+
+Che-Theia is currently the default integrated development environment (IDE) that a workspace loads with if no IDE is specified as a URL parameter as part of the URL for starting a new workspace.
+
+The URL parameter for specifying another supported IDE is `che-editor=__<editor_key>__`.
+
+The URL for starting a new workspace has the following syntax with the `che-editor=__<editor_key>__` parameter:
+
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?che-editor=__<editor_key>__
+----
+
+.`__<editor_key>__` values for supported IDEs
+|===
+| IDE | `__<editor_key>__` value | Note
+
+ifeval::["{project-context}" == "che"]
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+| `eclipse/che-theia/latest`
+| This is the default IDE with link:https://github.com/che-incubator/chectl/[chectl stable]. This IDE loads in a new workspace without entering this URL parameter.
+
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+| `eclipse/che-theia/next`
+| This is the default IDE with link:https://github.com/che-incubator/chectl/[chectl next]. This IDE loads in a new workspace without entering this URL parameter.
+endif::[]
+
+ifeval::["{project-context}" == "crw"]
+| link:https://github.com/eclipse-che/che-theia[Che-Theia]
+|`eclipse/che-theia/latest`
+| This is the default IDE: it loads in a new workspace without this URL parameter.
+endif::[]
+
+| link:https://coder.com/docs/code-server/[*code-server*]
+| `che-incubator/che-code/insiders`
+|
+
+| link:https://www.jetbrains.com/help/idea/discover-intellij-idea.html[*IntelliJ IDEA Community Edition*]
+| `che-incubator/intellij-community/latest`
+|
+
+| link:https://www.jetbrains.com/help/pycharm/quick-start-guide.html[*PyCharm*]
+| `che-incubator/pycharm/latest`
+|
+
+|===
+
+== URL parameter for starting duplicate workspaces
+
+Visiting a URL for starting a new workspace results in a workspace that is based on a combination of a devfile and a linked Git repository.
+
+In some situations, you may need to start one or more workspaces from the same URL for starting a new workspace, resulting in workspaces that are duplicates in terms of the same devfile and the same linked Git repository. The URL parameter for starting a duplicate workspace is `new`.
+
+The URL for starting a new workspace has the following syntax with the `new` parameter:
+
+[source,subs="+quotes,+attributes"]
+----
+{prod-url}#__<git_repository_url>__?new
+----
+
+NOTE: Attempting to reuse the devfile and linked Git repository without the `new` URL parameter to start a new workspace in addition to a workspace that already exists results in an error message.
```

Comment:
Would this message be clearer if it was composed of two or three sentences?

Reply:
@rolfedh, this is how I solved it:
```suggestion
NOTE: If you already have a workspace that you started using a URL, then visiting the URL again without the `new` URL parameter results in an error message.
```